README.adoc

```diff
@@ -25,24 +25,20 @@ Learn how to use MicroProfile Rest Client to invoke RESTful microservices over H
 
 == What you'll learn
 
-You will learn how to build a MicroProfile Rest Client to access remote RESTful services. MicroProfile Rest Client provides a type-safe way to invoke RESTful services with template interfaces.
+You will learn how to build a MicroProfile Rest Client to access remote RESTful services. You will create a template interface that maps to the remote service that you want to call.
+The MicroProfile Rest Client API automatically generates an instance of the template interface based on its annotations.
+Thus, you don't have to worry about all of the boilerplate code, such as setting up a client class, connecting to the remote server, or invoking the correct URI with the correct parameters.
 
 The application that you will be working with is an `inventory` service, which fetches and stores the system property information for different hosts.
 Whenever a request is made to retrieve the system properties of a particular host, the `inventory` service will create a client to invoke the `system`
 service on that host. The `system` service simulates a remote service in the application.
 
-You will learn how to create a template interface that maps to the remote service that you want to call.
-The MicroProfile Rest Client API automatically generates an instance of the template interface based on its annotations. Thus, you don't waste time on setting up the client and connecting with the remote service.
-
 You will instantiate the client and use it in the `inventory` service. You can choose from two different approaches, Context and Dependency Injection (CDI) with the help of MicroProfile Config or the RestClientBuilder method.
-When the CDI injection is available for you, and the base URL is static and able to configure, you can inject the client with CDI easily. When the CDI option is not available, use the more verbose `RestClientBuilder` method.
-In this guide, you will explore both methods to handle different scenarios for providing a valid base URL.
-
-You will handle various scenarios with the two methods:
+In this guide, you will explore both methods to handle two scenarios for providing a valid base URL:
 
- - With the CDI method, define the default base URL with the `localhost` host name in the configuration file and inject the client.
+ - When the base URL of the remote service is static and able to configure, define the default base URL with the `localhost` host name in the configuration file. Inject the client with CDI method.
```

Comment:
Not sure "with the `localhost` host name" is appropriate here; need to see the whole guide to determine. It may not be needed.